lib/streamlit/hello.py

```diff
@@ -14,57 +14,243 @@
 # limitations under the License.
 
 """A "Hello World" app."""
+from __future__ import division
 
 import streamlit as st
-import numpy as np
-import time
+import inspect
+from collections import OrderedDict
+import urllib
 
-
-def run():
-    st.title("Hello Streamlit!")
-    st.write(
+def intro():
+    st.markdown(
         """
-        This is an example **Streamlit** app.
+## Intro...
 
-        If this is your first time using Streamlit, welcome! You may want to
-        take a look at our
-        [tutorials and documentation](https://streamlit.io/docs) next.
-        And feel free to ask us questions any time at
-        [help@streamlit.io](mailto:help@streamlit.io).
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+"""
+    )
 
-        To celebrate this magic moment, below we're generating a bunch of
-        random numbers in a loop for around 10 seconds. Enjoy!
+
+def mapping_demo():
     """
-    )
+    This demo shows how to use
+    [`st.deck_gl_chart`](https://streamlit.io/docs/api.html#streamlit.deck_gl_chart)
+    to display geospatial data.
+    """
```

Comment:
If you're adding docstrings to these functions (not really required since they're private and very understandable anyway), please use the Numpy docstring format. In particular, the first sentence of the docstring should be a 1-line summary that starts at the `"""`.

Same with other functions below.